src/components/FormItem-HomeWork/FormItem-HomeWork.tsx

```diff
@@ -0,0 +1,32 @@
+import { ControllerFieldState, ControllerRenderProps } from "react-hook-form";
+import { IForm } from "../../sections/useForm-HomeWork";
+import cn from "classnames";
+import styles from "./FormItem.module.scss";
+
+interface IFormItem {
+    field: ControllerRenderProps<IForm, any>;
+    fieldState: ControllerFieldState;
+    labelText: string;
+}
+
+export const FormItemHomeWork = ({ field, fieldState, labelText }: IFormItem) => {
```

Comment:
Ideally, this cmp should be reusable with different types of inputs such as select, text input, radio, checkbox
so in ideal scenario, your input should be a separate cmp